lib/vagrant-proxyconf/action.rb

```diff
@@ -31,7 +31,7 @@ def self.configure(opts = {})
       def self.configure_after_provisoner
         Vagrant::Action::Builder.new.tap do |b|
           b.use Builtin::Call, IsEnabled do |env, b2|
-            next if !env[:result]
+            # next if !env[:result]
```

Comment:
Sorry, I don't get this. Why is the result not used? Is the `IsEnabled` action then needed at all?

Reply:
As a side note, maybe we should change the config to be more clear. Maybe replace the `config.proxy.enabled` with something like `config.proxy.state` which accepts values like `:configure` (default), `:clear` and `:ignore` (and their string versions so that env vars work, too). Dunno, what do you think?

Reply:
Let me test this and get back to you.  I think I remember commenting this out because if the proxy is disabled either globally or individually it would just "skip over" unconfiguring the proxy if it had previously been configured.  

Let me confirm this is still the case and if it is, i'll fix it in a commit and add a message as to why.

Thanks again for your help with and for asking questions.  I'm a big fan of code review and I'm always looking for ways to improve.  :)

Reply:
I don't think we can't break the behaviour of `config.proxy.enabled = false` without major version release. Adding a new configuration (and warning for the deprecating of the old one), or adding more possible values to the current one (with backward compatibility) is fine. The need for totally skipping the plugin (or parts of it) still exists.

Reply:
Works for me, thanks again for your help and for granting me access to do the things to the stuff.  I too don't want to spend to much more time with this.